GraphBulkImporter/Program.cs

```diff
@@ -0,0 +1,194 @@
+﻿namespace GraphBulkImporter
+{ 
+    using System;
+    using System.Configuration;
+    using System.Diagnostics;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using Microsoft.Azure.Documents;
+    using Microsoft.Azure.Documents.Client;
+    using Microsoft.Azure.CosmosDB.BulkExecutor;
+    using Microsoft.Azure.CosmosDB.BulkExecutor.BulkImport;
+    using Microsoft.Azure.CosmosDB.BulkExecutor.Graph;
+    using GraphBulkImporter.Models;
+
+    class Program
+    {
+        private static readonly string EndpointUrl = ConfigurationManager.AppSettings["EndPointUrl"];
+        private static readonly string AuthorizationKey = ConfigurationManager.AppSettings["AuthorizationKey"];
+        private static readonly string DatabaseName = ConfigurationManager.AppSettings["DatabaseName"];
+        private static readonly string CollectionName = ConfigurationManager.AppSettings["CollectionName"];
+        private static readonly int CollectionThroughput = int.Parse(ConfigurationManager.AppSettings["CollectionThroughput"]);
+
+        private static readonly ConnectionPolicy ConnectionPolicy = new ConnectionPolicy
+        {
+            ConnectionMode = ConnectionMode.Direct,
+            ConnectionProtocol = Protocol.Tcp
+        };
+
+        private static DocumentClient client;
+
+        public async static Task Main(string[] args)
+        {
+            Trace.WriteLine("Summary:");
+            Trace.WriteLine("--------------------------------------------------------------------- ");
+            Trace.WriteLine(String.Format("Endpoint: {0}", EndpointUrl));
```

Comment:
[nit] I see you using string interpolation in some placed and string.Format here. Interpolation ends up compiling down to string.Format anyways but might want to just make it more consistent